python/pyspark/profiler.py

```diff
@@ -142,11 +152,13 @@ class PStatsParam(AccumulatorParam):
     """PStatsParam is used to merge pstats.Stats"""
```

Comment:
This still has to be filled with full type, as per previous comments.

Reply:
Sorry, I missed your previous comment! Updated :smiley: